src/app/auth/login/login.component.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../../styles.scss";
+
+:host {
+  position: relative;
```

Comment:
I am not sure that you need to use in this case ::host modifier. It should be used in cases when it is really needed.
It is very simple case and i think you can do it without this modifier.